api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -177,10 +177,11 @@ echo "Inserting concept table data ... "
 bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 "INSERT INTO \`$WORKBENCH_PROJECT.$WORKBENCH_DATASET.concept\`
 (concept_id, concept_name, domain_id, vocabulary_id, concept_class_id, standard_concept,
-concept_code, count_value, prevalence, source_count_value)
-SELECT c.concept_id, c.concept_name, c.domain_id, c.vocabulary_id, c.concept_class_id, c.standard_concept, c.concept_code,
-0 as count_value , 0.0 as prevalence, 0 as source_count_value
-from \`$BQ_PROJECT.$BQ_DATASET.concept\` c"
+concept_code, count_value, prevalence, source_count_value, synonyms)
+select c.concept_id, c.concept_name, c.domain_id, c.vocabulary_id, c.concept_class_id, c.standard_concept, c.concept_code,
+0 as count_value , 0.0 as prevalence, 0 as source_count_value,concat(cast(c.concept_id as string),'\\|',c.vocabulary_id,':',c.concept_code,'\\|',c.concept_name,'\\|',string_agg(cs.concept_synonym_name,'\\|')) as synonyms
```

Comment:
So it looks like the delimiter is now \| instead of |?

That's OK, and this definitely reduces the chance of the delimiter appearing in a concept name or synonym, though it's still technically possible. If we wanted to avoid any possibility of this happening, what you'd do is escape occurrences of the delimiter in concept names or synonyms... so you could (for example) use | as the delimiter but replace all occurrences of | with || or whatever.

Anyway... I think this combination of characters is sufficiently unlikely to occur that we should be OK.